test/blackbox-tests/test-cases/pkg/dune

```diff
@@ -26,7 +26,7 @@
  (deps %{bin:strace}))
 
 (cram
- (deps %{bin:git} %{bin:awk} %{bin:cmp})
```

Comment:
Why is `git` being removed here?

Reply:
It's duplicated with a deceleration above. I only noticed because with no `git` we get two errors.